docs/testing.md

```diff
@@ -301,55 +336,50 @@ module.exports = {
 }
 ```
 
-You can learn more about each option above in the [Jest docs](https://jestjs.io/docs/configuration).
+You can learn more about each configuration option in the [Jest docs](https://jestjs.io/docs/configuration).
 
 **Handling stylesheets and image imports**
 
-These files aren't useful in tests but importing them may cause errors, so we will need to mock them. Create the mock files we referenced in the configuration above - `fileMock.js` and `styleMock.js` - inside a `__mocks__` directory:
+Styleheets and images aren't used in the tests but importing them may cause errors, so we will need to mock them. Create the mock files we referenced in the configuration above - `fileMock.js` and `styleMock.js` - inside a `__mocks__` directory:
 
 ```json
```

Comment:
```suggestion
```js
```

Reply:
Thank you for spotting that @stefanprobst 🙏🏼